tests/phpunit.xml.dist

```diff
@@ -1,13 +0,0 @@
-<phpunit bootstrap="./Bootstrap.php" colors="true">
```

Comment:
Possible rebase conflict? Or is this change intentional? See https://github.com/zendframework/ZendService_Twitter/commit/5320b6c1e8ba950556fd76144a694dd102049f1a